database/seeds/InventoriesTableSeeder.php

```diff
@@ -13,17 +13,17 @@ class InventoriesTableSeeder extends Seeder
     public function run()
     {
         $inventories = collect([
-            ['quantity' => '1', 'low' => 1, 'high' => 0],
-            ['quantity' => '3', 'low' => 1, 'high' => 0],
-            ['quantity' => '6', 'low' => 1, 'high' => 0],
-            ['quantity' => '7', 'low' => 1, 'high' => 0],
-            ['quantity' => '8', 'low' => 1, 'high' => 0],
-            ['quantity' => '5', 'low' => 1, 'high' => 0],
-            ['quantity' => '4', 'low' => 1, 'high' => 0],
-            ['quantity' => '3', 'low' => 1, 'high' => 0],
-            ['quantity' => '5', 'low' => 1, 'high' => 0],
-            ['quantity' => '77', 'low' => 0, 'high' => 1],
-            ['quantity' => '55', 'low' => 0, 'high' => 1],
+            ['id' => '1', 'ingredient_id' => '1', 'quantity' => '1', 'low' => 1, 'high' => 0],
```

Comment:
i know this was an issue with your table before merge but you can remove the ingred_id now i believe before merging

Reply:
Yeah this was just a work around for the time being so I could get the seeder to work properly, forgot to change it to use a factory like the master :X

Reply:
maybe do a git merge master into your branch